apps/public-docsite-v9/.storybook/docs-root-v9.css

```diff
@@ -0,0 +1,42 @@
+/* remove the docs wrapper bg to let page bg show through */
+/* remove unnecessary padding now that theme switcher is not taking up space */
+#docs-root .sbdocs-wrapper {
+  background: transparent !important;
+  padding-top: 0;
+}
+
+/* sb-show-main is missing during page transitions causing a page shift */
+/* todo: cleanup once we no longer inherit docs-root */
+.sb-show-main.sb-main-fullscreen,
+.sb-main-fullscreen {
+  margin: 0;
+  padding: 0;
+  display: block;
+}
+
+/* remove background from fluent provider to allow page bg to show through */
+#docs-root > .fui-FluentProvider {
+  background: transparent !important;
+}
+
+/* remove background from theme switcher */
+/* todo: remove override once we have a custom theme solution for website */
+#docs-root > .fui-FluentProvider > div {
```

Comment:
why does this override need to be done here ? this can be done in the code that contains the theme picker https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/src/DocsComponents/FluentDocsHeader.stories.tsx

Reply:
thought that the theme picker was part of a plugin, and didn't want to mess with other dev loops. Looks like I was quite wrong, and i'll move these updates all into the ThemePicker.stories.tsx file

Reply:
@ling1726 took care of moving those styles over